Makefile

```diff
@@ -3,15 +3,9 @@ include n.Makefile
 install:
 	obt install --verbose
 
-build: clean
-	obt build --js=./src/main.js --sass=./src/main.scss --buildFolder=./
-
-clean:
-	rm -f ./main.*;
-
```

Comment:
Don't know why these two were here but didn't make sense because the actual code was in ./main.*